### PR TITLE
EPMDJ-2344.Fixed incorrect content-type and examples of responses.

### PR DESCRIPTION
**Pull request by ViktoriiaLebedeva**

### CODE REVIEW

**Review by viktorbll — CHANGES_REQUESTED** [review]
Too much refactoring for such simple change.
The task is not about refactoring.
Change to empty content leads to information loss in some cases.

File: core/src/main/kotlin/endpoints/openapi/DrillAdminEndpoints.kt [hunk 1]
```diff
@@ -207,20 +201,16 @@ class DrillAdminEndpoints(override val kodein: Kodein) : KodeinAware {
...
+                    val (statusCode, response) = if (notificationId.isBlank()) {
                         notificationsManager.deleteAll()
-                        HttpStatusCode.OK to "All notifications successfully deleted"
+                        HttpStatusCode.OK to EmptyContent
                     } else {
-                        val (notificationId) = notificationIdObject
                         if (notificationsManager.delete(notificationId)) {
                             topicResolver.sendToAllSubscribed("/notifications")
-                            HttpStatusCode.OK to "Notification with id $notificationId successfully deleted"
+                            HttpStatusCode.OK to EmptyContent
```
Comment (viktorbll) [inline core/src/main/kotlin/endpoints/openapi/DrillAdminEndpoints.kt]: information is lost in this case

File: core/src/main/kotlin/endpoints/openapi/DrillAdminEndpoints.kt [hunk 2]
```diff
@@ -176,22 +175,17 @@ class DrillAdminEndpoints(override val kodein: Kodein) : KodeinAware {
...
-                        ok<String>(
-                            example("result", "All notifications successfully read"),
-                            example("result", "Notification with such id successfully read")
-                        ),
-                        notFound()
+                        ok<Unit>(), notFound()
                     )
-                post<Routes.Api.ReadNotification, NotificationId>(notificationResponds) { _, notificationIdObject ->
+                post<Routes.Api.ReadNotification, NotificationId>(notificationResponds) { _, (notificationId) ->
                     logger.info { "Read notification" }
```
Comment (LikhanovMaxim) [inline core/src/main/kotlin/endpoints/openapi/DrillAdminEndpoints.kt]: This is not enough informative log, we should add 'notificationId'

**Review by LikhanovMaxim — APPROVED** [review]
Approved in case that we have created a task with redesign endpoints: notifications/read & notifications/delete

File: core/src/main/kotlin/endpoints/openapi/DrillAdminEndpoints.kt [hunk 3]
```diff
@@ -182,13 +178,12 @@ class DrillAdminEndpoints(override val kodein: Kodein) : KodeinAware {
                         ),
                         notFound()
                     )
-                post<Routes.Api.ReadNotification, NotificationId>(notificationResponds) { _, notificationIdObject ->
-                    logger.info { "Read notification" }
-                    val (statusCode, response) = if (notificationIdObject.notificationId.isBlank()) {
+                post<Routes.Api.ReadNotification, NotificationId>(notificationResponds) { _, (notificationId) ->
```
Comment (viktorbll) [inline core/src/main/kotlin/endpoints/openapi/DrillAdminEndpoints.kt]: revert this

File: core/src/main/kotlin/endpoints/openapi/DrillAdminEndpoints.kt [hunk 4]
```diff
@@ -212,12 +207,11 @@ class DrillAdminEndpoints(override val kodein: Kodein) : KodeinAware {
                         ),
                         notFound()
                     )
-                post<Routes.Api.DeleteNotification, NotificationId>(notificationResponds) { _, notificationIdObject ->
-                    val (statusCode, response) = if (notificationIdObject.notificationId.isBlank()) {
+                post<Routes.Api.DeleteNotification, NotificationId>(notificationResponds) { _, (notificationId) ->
```
Comment (viktorbll) [inline core/src/main/kotlin/endpoints/openapi/DrillAdminEndpoints.kt]: revert this

**Pull request merged**